targets/targets.json

```diff
@@ -3630,12 +3630,19 @@
             }
         },
         "overrides": {
-            "lse_available": 0
+            "lse_available": 0,
+            "boot-stack-size": "0x400",
+            "tickless-from-us-ticker": true
         },
         "detect_code": [
             "0775"
         ],
-        "default_lib": "small",
+        "supported_c_libs": {
+            "arm": ["std", "small"],
+            "gcc_arm": ["std", "small"],
+            "iar": ["std"]
+        },
+        "c_lib": "small",
```

Comment:
Means that IAR is no more supported ?

Reply:
IAR should still be supported thanks to the following PR: https://github.com/ARMmbed/mbed-os/pull/12665